File: angular-frontend/src/app/demo/api/inventory_header.ts
import { Company } from "./company"
import { Inventory_detail } from "./inventory_detail"
import { User } from "./users"

export interface Inventory_header {
    ID?: number,
    Inv_head_date?: Date
    Company_id?: number
    Users_id?: number
    Inventory_details?: Inventory_detail[] 
    Company?: Company
    Users?:User
}

File: gin-backend/pkg/common/models/inventory_header.go
package models

import (
	"time"

	"gorm.io/gorm"
)

type Inventory_header struct {
	gorm.Model
	Inv_head_date     time.Time
	Company_id        uint
	Users_id          uint
	Inventory_details []Inventory_detail
	Company           *Company
	Users             *Users
}
